### DISCUSSION TITLE
桑基图效果

### DISCUSSION THREAD

**Discussion by yuanAIhan** [antvis/G2Plot#3569] [Q&A]
![image](https://user-images.githubusercontent.com/80769611/226513960-e40bb197-ade2-456b-83d2-fefcf7d27869.png)
如图所示，鼠标移入节点时，如何显示总计，并下一层所有数据流高亮显示

💻 Link to minimal reproduction
Please provide a link by forking these links [g2plot](https://codesandbox.io/s/g2plot-reproduction-template-gyej1) or GitHub repo. What is a minimal reproduction, and why is it required?

🏞 Expected result
期望效果，如图所示
![image](https://user-images.githubusercontent.com/80769611/226513999-81b33877-1f4d-4089-808a-7f7f30a1412c.png)

这个问题会解决吗？

**Comment by hustcc** [comment]
这个不会在 G2Plot 中内置，算是一个定制需求，可以用自定义交互的形式实现，可以参考 G2Plot 代码中目录中的 sankey，里面有自定义拖拽节点的交互。
